### PR TITLE
Fixed fatal error when fetching group context when the route object has not been set.

### LINKED ISSUE #345: Call to a member function getPath() on null when group context is requested and no route has been set
I bumping into this issue when writing some tests.
I'm trying to fetch the group context quite early on in the request and `$this->routeMatch->getRouteObject()` is returning `NULL` which results in the following error:

```
Error: Call to a member function getPath() on null
Drupal\og\OgRouteGroupResolverBase->getContentEntity()() (Line: 92)
```

### PR DESCRIPTION
**Pull request by geertvd**
Fixed #345

### CODE REVIEW

File: src/OgRouteGroupResolverBase.php [hunk 1]
```diff
@@ -88,6 +88,10 @@ public static function create(ContainerInterface $container, array $configuratio
    *   The entity, or NULL if we are not on a content entity path.
    */
   protected function getContentEntity() {
+    $route = $this->routeMatch->getRouteObject();
+    if (!$route) {
+      return NULL;
+    }
     // Check if we are on a content entity path.
     $path = $this->routeMatch->getRouteObject()->getPath();
```
Comment (zerolab) [inline src/OgRouteGroupResolverBase.php]: should this not become `$route->getPath()` ?

Comment (geertvd) [reply src/OgRouteGroupResolverBase.php]: Yes, fixed that

**Comment by zerolab** [comment]
LGTM

**Comment by amitaibu** [comment]
Thanks guys

**Pull request merged**